modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -373,6 +373,149 @@ func (suite *TendermintTestSuite) TestCheckHeaderAndUpdateState() {
 	}
 }
 
+func (suite *TendermintTestSuite) TestUpdateState() {
+	var (
+		path                  *ibctesting.Path
+		clientMessage         exported.ClientMessage
+		pruneHeight           clienttypes.Height
+		updatedClientState    *types.ClientState    // TODO: retrieve from state after 'UpdateState' call
+		updatedConsensusState *types.ConsensusState // TODO: retrieve from state after 'UpdateState' call
+	)
+
+	testCases := []struct {
+		name      string
+		malleate  func()
+		expResult func()
```

Comment:
I find it useful to add a expected results callback for test cases which need to check unique state changes

Reply:
Nice, I could see this being a useful approach for testing other state updates. ICS29 tests come to mind! :)